compiler/ballerina-lang/src/main/java/io/ballerina/projects/ModuleContext.java

```diff
@@ -133,6 +137,13 @@ ModuleName moduleName() {
         return moduleDescriptor.name();
     }
 
+    byte[] getBirBytes() {
+        if (birBytes == null) {
+            birBytes = new BIRBinaryWriter(this.bLangPackage.symbol.bir).serialize();
```

Comment:
this.bLangPackage can be `null`, let's use `getBLangPackageOrThrow` so we get a better error message.